utils/python/tests/scripts_regression_tests.py

```diff
@@ -1635,7 +1663,7 @@ def _main_func():
 
     CIME.utils.handle_standard_logging_options(args)
 
-    unittest.main(verbosity=2)
+    unittest.main(verbosity=2, catchbreak=True)
```

Comment:
catchbreak allows you to abort the unit test run with ctrl-C, and then get a full report of the tests that have failed so far. I found this useful, if I got some failures in the very first tests - so I could see a full report without having to wait for the entire scripts_regression_tests to finish.